CHANGELOG.md

```diff
@@ -1,3 +1,57 @@
+* **slide-toggle:** remove webkit tap highlight ([#12708](https://github.com/angular/material2/issues/12708)) ([3ce4e8d](https://github.com/angular/material2/commit/3ce4e8d))
+* **slider:** thumb label blending in with background in high contrast mode ([#12606](https://github.com/angular/material2/issues/12606)) ([a4a79ea](https://github.com/angular/material2/commit/a4a79ea))
+* **stepper:** focus lost if focus is inside stepper while changing step ([#12761](https://github.com/angular/material2/issues/12761)) ([fc537af](https://github.com/angular/material2/commit/fc537af))
+* **stepper:** handle removing a step before the current one ([#11813](https://github.com/angular/material2/issues/11813)) ([0fcdae4](https://github.com/angular/material2/commit/0fcdae4)), closes [#11791](https://github.com/angular/material2/issues/11791)
+* **stepper:** improved alignment for step icons ([#12703](https://github.com/angular/material2/issues/12703)) ([41d0196](https://github.com/angular/material2/commit/41d0196)), closes [#12696](https://github.com/angular/material2/issues/12696)
+* **table:** errors when rendering table with sticky elements on the server ([#12095](https://github.com/angular/material2/issues/12095)) ([8956d1d](https://github.com/angular/material2/commit/8956d1d)), closes [#12094](https://github.com/angular/material2/issues/12094)
+* **table:** extra elements throwing off table alignment ([#12645](https://github.com/angular/material2/issues/12645)) ([13f1c6e](https://github.com/angular/material2/commit/13f1c6e)), closes [#11165](https://github.com/angular/material2/issues/11165)
+* **tabs:** disable focus overlay for touch focus ([#12249](https://github.com/angular/material2/issues/12249)) ([d3af441](https://github.com/angular/material2/commit/d3af441)), closes [#12247](https://github.com/angular/material2/issues/12247)
+* **tabs:** only target direct descendants with mat-stretch-tabs ([#12198](https://github.com/angular/material2/issues/12198)) ([6bb0ffe](https://github.com/angular/material2/commit/6bb0ffe)), closes [#12196](https://github.com/angular/material2/issues/12196)
+* **tooltip:** opening after click on android ([#12250](https://github.com/angular/material2/issues/12250)) ([a6b8a06](https://github.com/angular/material2/commit/a6b8a06)), closes [#12223](https://github.com/angular/material2/issues/12223)
+* ensure all components work with ES2015 in jit mode. ([#12759](https://github.com/angular/material2/issues/12759)) ([0c4a1b4](https://github.com/angular/material2/commit/0c4a1b4)), closes [/github.com/angular/angular/pull/22356#issuecomment-387756794](https://github.com//github.com/angular/angular/pull/22356/issues/issuecomment-387756794) [#9329](https://github.com/angular/material2/issues/9329)
```

Comment:
This link seems to be broken. We should fix that.

Reply:
Going to merge for now, let's fix in a follow up